src/typechecker/types.lisp

```diff
@@ -217,17 +199,7 @@
 (defvar *boolean-type*      (make-tycon :name 'coalton:Boolean      :kind +kstar+))
 (defvar *unit-type*         (make-tycon :name 'coalton:Unit         :kind +kstar+))
 (defvar *char-type*         (make-tycon :name 'coalton:Char         :kind +kstar+))
-(defvar *u8-type*           (make-tycon :name 'coalton:U8           :kind +kstar+))
```

Comment:
Were these variables just unused?

Reply:
From a cursory search, I don't see them ever being used

Reply:
Yes, unused. these styles wired-in type constructors are used in two places: in the manual definition of early types in typechecker/environment.lisp and in typechecker/context-reduction.lisp, to resolve ambiguous lisp literals -- other access is conventional, through the environment.

Many early types, notably char and string, look like they could be ejected without much effort from the compiler, and defined directly in the standard library.